src/components/Request.js

```diff
@@ -19,14 +19,18 @@ class RequestMaker extends Component {
 
         // Callback executed after the network succeeds.
         then: PropTypes.func,
+
+        // Callback executed after the network requests ends, no matte the result.
+        finally: PropTypes.func,
     }
 
     static defaultProps = {
         then: () => undefined,
+        finally: () => undefined,
     }
 
     static get endpoint() {
-        return 'https://kudosapp-production.herokuapp.com/'
+        return 'https://kudosapp-staging.herokuapp.com/'
```

Comment:
mmm przydałoby się to z propsów brać jednak ;-)

Reply:
To już jest w #56, więc tutaj tylko zhardkodowałem.

Reply:
Co w sumie jest głupie, bo zepsuje deploy. 🤦‍♂️